### ISSUE TITLE
Delete records

### ISSUE THREAD

**Issue by drimpact** [impactasaurus/app#8]
Allow users to delete meetings associated with a beneficiary

**Comment by drimpact** [comment]
Requires a list of meetings to be shown, similar to #31 and #18

**Comment by drimpact** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Assessment management](https://trello.com/c/UQUYQHKI/26-assessment-management)

**Comment by drimpact** [comment]
Blocked by #108

**Comment by drimpact** [comment]
This is no longer needed with impactasaurus/server#57

**Comment by drimpact** [comment]
Users are asking for this capability

**Comment by drimpact** [comment]
Expanded the scope of #141, so this is not needed
